docs/preview/features/correlation.md

```diff
@@ -22,9 +22,10 @@ This correlation is based on the `RequestId` and `X-Transaction-ID` HTTP request
 
 ## How This Works
 
-When an application is configured to use the default configuration of the correlation, each HTTP response will get an extra header called `RequestId` containing an unique identifier to distinguish between requests/responses.
+When an application is configured to use the default configuration of the correlation, each HTTP **response** will have an extra header called `RequestId` containing an unique identifier to distinguish between requests/responses. This ID will act as the parent ID for all telemetry that comes after and uses the [HTTP Correlation](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Diagnostics.DiagnosticSource/src/HttpCorrelationProtocol.md) to extract the most recent ID.
```

Comment:
```suggestion
When an application is configured to use the default configuration of the correlation, each HTTP **response** will have an extra header called `RequestId` containing an unique identifier to distinguish between requests/responses. This ID will be used by the [HTTP Correlation](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Diagnostics.DiagnosticSource/src/HttpCorrelationProtocol.md) as the parent ID for all the requests that are triggered by this operation.
```

Reply:
'requests' seems misleading, no? In what contexts are there more than 1 requests linked to a single dependency?

Reply:
In any case, I would want us to add a user guide for observability that explains fully the service-to-service correlation and how Web API and Messaging can be used in combination for this.

Reply:
Indeed, I was doubting how to write it.
I actually meant 'requests initiated by calls to dependencies from this operation'.   So, your operation has a dependency on service X, and initiates a 'request' to Service X.   Service X will receive that request, and will log it as a request.

Reply:
Maybe in this context, we shouldn't use 'request' and 'dependency' in a not telemetry type way.
I mean, not say: '... has a dependency on...', '...sends request too...', ... bc it becomes really confusing. Since it's multiple services, the viewpoint also alters the name. In the sending service it is a dependency, in the receiving service it is a request. Maybe a visual can help here.

Reply:
Rewrote the entire section and added a visual here: https://github.com/arcus-azure/arcus.webapi/pull/345/commits/92657efdc2aa14b157ba08761c31945565a5e424

Reply:
Great! Thanks.